R/geom-bin2d.r

```diff
@@ -31,7 +31,7 @@ geom_bin2d <- function(mapping = NULL, data = NULL,
     data = data,
     mapping = mapping,
     stat = stat,
-    geom = GeomTile,
+    geom = GeomBind2d,
```

Comment:
why do you change this?

Reply:
I did this because I had to add a method which was supposed to be applied on `GeomBind2d` geom only.

Reply:
ok that makes sense but shouldn't it be Bin2d? (not Bind2d) (no extra d before 2)

Reply:
Oh! I'm so so sorry! How could I not see that!